ow_system_plugins/admin/controllers/permissions.php

```diff
@@ -112,6 +112,16 @@ public function index()
                 }
                 else
                 {
+                    $adminEmail = OW::getUser()->getEmail();
+                    $senderMail = $config->getValue('base', 'site_email');
+                    $mail = OW::getMailer()->createMail();
+                    $mail->addRecipientEmail($adminEmail);
+                    $mail->setSender($senderMail);
+                    $mail->setSenderSuffix(false);
+                    $mail->setSubject('Site password');
+                    $mail->setTextContent("Your password: " . $data['password']);

```

Comment:
You should use localization system. It looks like this: 
```php
$mail->setTextContent(
    OW::getLanguages()->text('sms_verify', 'your_password', array('password' => $data['password']))
);
```

The same note for previous code.